test/acceptance/tests/hcp/hcp_test.go

```diff
@@ -595,3 +595,69 @@ func restoreConsulState(t *testing.T, consul *api.Client, state ConsulState) err
 	}
 	return nil
 }
+
+func TestAuditLogging(t *testing.T) {
+	// read the configuration from the setup-terraform dir.
+	var cfg HCPTestConfig
+	require.NoError(t, UnmarshalTF(setupDir, &cfg))
+
+	// generate input variables to the test terraform using the config.
+	ignoreVars := []string{"ecs_cluster_1_arn", "ecs_cluster_2_arn", "token"}
```

Comment:
As of [a few hours ago](https://github.com/hashicorp/terraform-aws-consul-ecs/pull/126), we now only run HCP tests on main and release branches (sorry 😃) and not on all PR branches. That's because HCP clusters take a while to spin up and tear down, and we want our PR checks to run faster.

That means you'll need to make the following change, so that this test is skipped if for PRs. The test case will be run in our CI jobs after it merges into main.

```suggestion
	cfg := parseHCPTestConfig(t)

	// generate input variables to the test terraform using the config.
	ignoreVars := []string{"ecs_cluster_1_arn", "ecs_cluster_2_arn", "token", "enable_hcp"}
```